encoders/nlp/CLIPTextEncoder/__init__.py

```diff
@@ -0,0 +1,44 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+import torch
+import clip
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+from jina.executors.devices import TorchDevice
+
+class CLIPTextEncoder(BaseTorchEncoder):
+    """
+    Encodes data from a `np.ndarray` (of strings) of length `BatchSize` into
+    a `np.ndarray` of shape `Batchsize x EmbeddingDimension`. 
+
+    Internally, :class:`CLIPImageEncoder` wraps the `CLIP` model from https://github.com/openai/CLIP
+
+    :param model_name: the name of the model. Supported models include ``ViT-B/32`` and ``RN50``.
```

Comment:
We need to add docstring for the *args and **kwargs too to be consistent, could you also add those?

Reply:
yes, done